cylc/uiserver/resolvers.py

```diff
@@ -189,7 +189,10 @@ def __init__(self, data, log, **kwargs):
     # Mutations
     async def mutator(self, info, *m_args):
         """Mutate workflow."""
-        _, w_args, _ = m_args
+        req_meta = {}
+        _, w_args, _, _ = m_args
```

Comment:
Tuple unpacking has gained an extra member by mistake?
```suggestion
        _, w_args, _ = m_args
```

Reply:
I know the test is failing here due to this but you should find that if you have the sibling checked out on cylc-flow then the test will pass (with the correct unpacking). The unpacking accounts for the meta.

Reply:
Oh, it's also done in https://github.com/cylc/cylc-flow/pull/4522. It's generally a good idea to have the same people review sibling PRs

Reply:
I'm tempted to ask GH for a multi-repo PR feature - surely we aren't the only people doing this sort of thing....

Reply:
It seems strange GH actions can't check out different branches. Maybe something to look into when time permits.

Reply:
Kinda doable, we could put a special comment in the PR description that tells GHA what base branch to use, however, the failing tests are also a good indicator not to merge until the dependent branch is in.